configure.ac

```
@@ -1878,6 +1879,24 @@ AS_CASE([$enable_ocamltest,OCAML__DEVELOPMENT_VERSION],
   [yes,*|,true],[ocamltest='ocamltest'],
   [ocamltest=''])
 
+AS_IF([test x"$ocamltest" = "xocamltest"],
```

Up to you, but I think this is slightly less brittle?
```suggestion
AS_IF([test -n "$1" ],
```